blackjack.rb

```diff
@@ -93,6 +105,13 @@ def player_hit_or_stay(current_hand)
     end
   end
 
+#cpu doesn't handle aces (unless I missed it).
+# I did something like this, but it assumes ace will auto change so you don't bust
+# if hand.any?{ |c| c.face == 'A' }
```

Comment:
Is this referring to the dealer's hand? Hands are an object in my project, so this would be dealah.hands_array[0].cards.any?, no?